fusionfrontend/src/app/ecosystem/components/content/asset-type-template/asset-type-template-list/asset-type-template-list.component.ts

```diff
@@ -51,20 +57,67 @@ export class AssetTypeTemplateListComponent extends BaseListComponent implements
   public createWizardRef: DynamicDialogRef;
   public assetTypeTemplateForm: FormGroup;
 
+  assetTypeTemplates: AssetTypeTemplate[];
+  displayedAssetTypeTemplates: AssetTypeTemplate[];
+  filteredAssetTypeTemplates: AssetTypeTemplate[];
+  searchedAssetTypeTemplates: AssetTypeTemplate[];
+
+  activeListItem: AssetTypeTemplate;
+
+  public menuType: ItemOptionsMenuType[];
+
+
+  private updateWizardRef: DynamicDialogRef;
+  private warningDialogRef: DynamicDialogRef;
+
+  tableFilters: FilterOption[] = [
+    { filterType: FilterType.DROPDOWNFILTER, columnName: 'Version', attributeToBeFiltered: 'publishedVersion' },
+    { filterType: FilterType.DATEFILTER, columnName: 'Publish date', attributeToBeFiltered: 'publishedDate' },
+    { filterType: FilterType.DROPDOWNFILTER, columnName: 'Status', attributeToBeFiltered: 'publicationState' }];
+
   constructor(
     public route: ActivatedRoute,
     public router: Router,
     public assetTypeTemplateQuery: AssetTypeTemplateQuery,
     public assetTypeTemplateService: AssetTypeTemplateService,
-    public dialogService: DialogService) {
-      super(route, router, assetTypeTemplateQuery, assetTypeTemplateService);
+    public dialogService: DialogService,
+    public confirmationService: ConfirmationService) {
      }
 
   ngOnInit() {
-    super.ngOnInit();
+    this.menuType = [ItemOptionsMenuType.DELETE];
+    if (this.assetTypeTemplates$ === undefined) {
```

Comment:
also check for null?  If you check for == null it automatically includes undefined